Wikipedia/Code/UIImage+WMFStyle.h

```diff
@@ -9,4 +9,9 @@
 
 + (instancetype)wmf_imageFromColor:(UIColor*)color;
 
+// Can directly use 'imageFlippedForRightToLeftLayoutDirection' once iOS 8 is no longer supported.
+- (UIImage*)wmf_imageFlippedForRTLLayoutDirectionIfAtLeastiOS9 WMF_DEPRECATED_WHEN_DEPLOY_AT_LEAST_9;
```

Comment:
w00t deprecation macros, FWIW though: using `wmf_isOperatingSystemVersionLessThan9_0_0` is already annotated w/ the same macro, so anything using it will automatically be deprecated once we drop iOS 8